cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/ScheduleTaskRunner.java

```diff
@@ -80,7 +80,8 @@ public ScheduleTaskRunner(Store store, ProgramLifecycleService lifecycleService,
     String scheduleName = systemOverrides.get(ProgramOptionConstants.SCHEDULE_NAME);
     ApplicationSpecification appSpec = store.getApplication(programId.getApplication());
     if (appSpec == null || appSpec.getSchedules().get(scheduleName) == null) {
-      throw new TaskExecutionException(UserMessages.getMessage(UserErrors.PROGRAM_NOT_FOUND), false);
```

Comment:
The `PROGRAM_NOT_FOUND` message also expects string interpolation. This was leaving the '%s' in the error message, without replacing it with the program.